### ISSUE TITLE
Tweaks for advice on setting initial enum value explicitly

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/Raizlabs-Cocoa-Style#21]
The style guide currently has an example where the initial value of an enum is set to `-1`. However, I prefer to set it to `0`. Here's why:

``` objc
typedef NS_ENUM(NSInteger, MyEnumType) {
    MyEnumTypeUnknown = 0,
    MyEnumTypeFoo,
    MyEnumTypeBar
}
```

If you have a property of this type, it will be initialized with a value of `0`, which is the unknown value. If you try to unwrap it from an `NSNumber` that happened to be `nil`, it will also be `0`, which is probably want you want. Why not just let the compiler set the first value to `0`, which it will do if you don't set it explicitly? Because if someone comes along and adds another value above the `Unknown` value, your assumption that that value would be equal to `0` is then broken.

**Comment by jkaufman** [comment]
Agreed, `0` should be the default recommendation. Anyone enumerating from `-1` should know why.
